pageserver/src/tenant/storage_layer/delta_layer.rs

```diff
@@ -946,6 +947,37 @@ impl DeltaLayerInner {
         Ok(planner.finish())
     }
 
+    fn get_min_read_buffer_size(
+        planned_reads: &[VectoredRead],
+        read_size_soft_max: usize,
+    ) -> usize {
+        let largest_read = match planned_reads.iter().max_by_key(|read| read.size()) {
+            Some(largest_read) => largest_read,
```

Comment:
this can be let else

Reply:
Good point: https://github.com/neondatabase/neon/pull/7259/commits/54a7c1b62b8599793a99c617ee48c324aa13b83e